packages/atlas/src/views/global/YppLandingView/YppHero.tsx

```diff
@@ -47,6 +47,8 @@ const getButtonText = (variant: YppStatus) => {
       return 'Sign up now'
     case 'no-channel':
       return 'Create channel & sign up'
+    case 'connect-wallet':
+      return 'Connect wallet'
```

Comment:
I checked the design and I think this is supposed to be `Sign up now` as well? 

Check here:  https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1774%3A115618&t=X01XdRaPbcksmb7b-4